zipline/data/us_equity_loader.py

```diff
@@ -179,50 +195,25 @@ def _ensure_sliding_window(
         except KeyError:
             pass
 
-        # Handle case where data is request before the start of data available
-        # in the daily_reader. In that case, prepend nans until the start
-        # of data.
-        pre_array = None
-        if start < self._daily_reader._calendar[0]:
-            start_ix = 0
-            td = self.env.trading_days
-            offset = td.get_loc(start) - td.get_loc(
-                self._daily_reader._calendar[0])
-            if end < self._daily_reader._calendar[0]:
-                fill_size = size
-                end_ix = 0
-            else:
-                pre_slice = self._calendar.slice_indexer(start, end)
-                fill_size = pre_slice.stop - pre_slice.start
-                end_ix = self._calendar.get_loc(end)
-            if field != 'volume':
-                pre_array = full((fill_size, 1), nan)
-            else:
-                pre_array = full((fill_size, 1), 0)
-        else:
-            offset = 0
-            start_ix = self._calendar.get_loc(start)
-            end_ix = self._calendar.get_loc(end)
+        start = dts[0]
+
+        offset = 0
```

Comment:
`offset` is only ever assigned 0, then passed into `SlidingWindow` below - can we get rid of it?